tests/CMakeLists.txt

```diff
@@ -295,59 +295,56 @@ if(WITH_PYTHON)
     TEST_FILES restarting/from_6.3.13/StorefrontTestRestart-1.txt
     restarting/from_6.3.13/StorefrontTestRestart-2.txt)
   add_fdb_test(
-    TEST_FILES restarting/from_6.3.13_until_7.2.0/DrUpgradeRestart-1.txt
-    restarting/from_6.3.13_until_7.2.0/DrUpgradeRestart-2.txt)
+    TEST_FILES restarting/from_6.3.13_until_71.3.0/DrUpgradeRestart-1.txt
+    restarting/from_6.3.13_until_71.3.0/DrUpgradeRestart-2.txt)
   add_fdb_test(
-    TEST_FILES restarting/from_7.0.0_until_7.1.0/UpgradeAndBackupRestore-1.toml
-    restarting/from_7.0.0_until_7.1.0/UpgradeAndBackupRestore-2.toml)
+    TEST_FILES restarting/from_7.0.0_until_71.2.0/UpgradeAndBackupRestore-1.toml
+    restarting/from_7.0.0_until_71.2.0/UpgradeAndBackupRestore-2.toml)
   add_fdb_test(
-    TEST_FILES restarting/from_7.1.0_until_7.2.0/ConfigureTestRestart-1.toml
```

Comment:
I'm not 100% sure what's right for these versions, i think it depends on the test.
For example, snowflake 71.2 was cut somewhere between apple 7.1 and apple 7.2, whereas snowflake 7.1 ~= apple 7.1 in terms of cut point
So, likely all of the tests that are "from 7.1" should still be "from 7.1"
But if the test is "from 7.2" then it should either be "from 71.2" or "from 71.3" depending on when it was added/if it was included in the snowflake/release-71.2 branch as "from 7.2" or not.